### ISSUE TITLE
Special case locator for "." and ".."?

### ISSUE THREAD

**Issue by waywardmonkeys** [dylan-lang/opendylan#722]
Should converting `"."` and `".."` to a `<file-system-locator>` be special cased to do the expected thing?

**Comment by cgay** [comment]
I agree with BarAgent on this.  "." means "." until you ask to resolve it.  If you mean `working-directory()` then say that instead.

I've added this test to `test-resolve-locator` in my current sources.
`assert-equal(working-directory(), resolve-locator(as(<file-system-locator>, ".")));`

**Comment by cgay** [comment]
In fact, accepting strings at all for file operations seems a little suspect to me.  (I get that it's really convenient in the REPL or whatever.)  It results in potential messes like this: 

<img width="504" alt="Image" src="https://github.com/user-attachments/assets/4cabd5a6-5de1-45d9-ad05-86628b311abd" />
